File: ProcessFiles/ProcessFiles/Address.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProcessFiles
{
    [Serializable]
    public class Address
    {
	//@SerializedName("Street1")
        private String _street1;

        public String Street1
        {
            get { return _street1; }
            set { _street1 = value; }
        }
	
	//@SerializedName("Street2")
        private String _street2;

        public String Street2
        {
            get { return _street2; }
            set { _street2 = value; }
        }
	
	//@SerializedName("City")
        private String _city;

        public String City
        {
            get { return _city; }
            set { _city = value; }
        }
	
	//@SerializedName("State")
        private String _state;

        public String State
        {
            get { return _state; }
            set { _state = value; }
        }
	
	//@SerializedName("PostalCode")
        private String _postalcode;

        public String Postalcode
        {
            get { return _postalcode; }
            set { _postalcode = value; }
        }
	
	//@SerializedName("Country")
        private String _country;

        public String Country
        {
            get { return _country; }
            set { _country = value; }
        }
    }
}

File: NotifyToProcessFiles/src/org/param/processfiles/Address.java
package org.param.processfiles;

import java.io.Serializable;

import com.google.gson.annotations.SerializedName;

public class Address implements Serializable{
	@SerializedName("Street1")
	private String _street1;
	
	@SerializedName("Street2")
	private String _street2;
	
	@SerializedName("City")
	private String _city;
	
	@SerializedName("State")
	private String _state;
	
	@SerializedName("PostalCode")
	private String _postalcode;
	
	@SerializedName("Country")
	private String _country;

	public String get_street1() {
		return _street1;
	}

	public void set_street1(String _street1) {
		this._street1 = _street1;
	}

	public String get_street2() {
		return _street2;
	}

	public void set_street2(String _street2) {
		this._street2 = _street2;
	}

	public String get_city() {
		return _city;
	}

	public void set_city(String _city) {
		this._city = _city;
	}

	public String get_state() {
		return _state;
	}

	public void set_state(String _state) {
		this._state = _state;
	}

	public String get_postalcode() {
		return _postalcode;
	}

	public void set_postalcode(String _postalcode) {
		this._postalcode = _postalcode;
	}

	public String get_country() {
		return _country;
	}

	public void set_country(String _country) {
		this._country = _country;
	}

}
